analyze_ipa.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"archive/zip"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+const (
+	contentsFolder = "contents"
```

Comment:
it would be better to uncompress the .ipa file to a tmp directory. what is the current directory already contains a `contents` directory?